### PR TITLE
add semicolons to appease a brittle parser

### LINKED ISSUE #1104: Some bslib SCSS causes brittle parsers to fail
Quarto needs to do some static analysis on SCSS files (yeah, laugh away), and this requires parsing the SCSS source. The best existing JS parser is relatively brittle and fails on some of the SCSS in bslib. On `main`, these are:

https://github.com/rstudio/bslib/blob/b447829fdde48e37df24a3497bc93c3d1643cf5b/inst/components/scss/value_box.scss#L88

https://github.com/rstudio/bslib/blob/b447829fdde48e37df24a3497bc93c3d1643cf5b/inst/components/scss/value_box.scss#L96

https://github.com/rstudio/bslib/blob/2b4bfc5a4d2e40b48338a38361ae8d453001edf8/inst/components/scss/sidebar.scss#L311

I'm not sure if the SCSS is actually invalid but most parsers accept the lack of terminating `;`, or if this is a bug in `scss-parser`. Nevertheless, it would be super helpful to add semicolons there if possible.

### PR DESCRIPTION
**Pull request by cscheid**
This closes #1104.

## Pull Request

Before you submit a pull request, please ensure you've completed the following che.klist

- [x] Ensure there is an already open and relevant [GitHub issue](https://github.com/rstudio/bslib/issues/new) describing the problem in detail and you've already received some indication from the maintainers that they are welcome to a contribution to fix the problem. This helps us to prevent wasting anyone's time. 

- [x] This project uses [roxygen2 for documentation](http://r-pkgs.had.co.nz/man.html). If you've made changes to documentation, run `devtools::document()`.

- [x] Run `devtools::check()` (or, equivalently, click on Build->Check Package in the RStudio IDE) to make sure your change did not add any messages, warnings, or errors.
    * Note there is a decent chance that some tests were already failing before your changes. Just make sure you haven't introduced any new ones.
    
- [x] Ensure your code changes follow the style outlined in http://r-pkgs.had.co.nz/style.html

- [x] Add an entry to NEWS.md concisely describing what you changed.

> - [ ] Add unit tests in the tests/testthat directory.

I haven't done that, because I'm not sure how you're testing SCSS syntax here. Happy to do so if someone points me in the right direction.

### CODE REVIEW

File: NEWS.md
Comment (cpsievert) [inline NEWS.md:4]:
```suggestion
* Added terminating semicolons to declarations in `value_box.scss` and `sidebar.scss` to prevent issues with parsers that require them. (@cscheid, #1104)
```

**Review by cpsievert — APPROVED** [review]
Thanks! Seems ok to me, but I'll let @gadenbuie decide before merging

File: NEWS.md
Comment (gadenbuie) [reply NEWS.md:4]: No offense @cscheid but I don't think this is a newsworthy change :smile

**Review by gadenbuie — APPROVED** [review]
LGTM! This should have net-zero effect on anything using bslib, these semi-colons will be stripped by libsass anyway.

File: NEWS.md
Comment (gadenbuie) [inline NEWS.md:4]:
```suggestion
```

Comment (cscheid) [reply NEWS.md]: I didn't know your criteria, my bad!

**Pull request merged**
